### ISSUE TITLE
Allow FROM instructions to have either one or three arguments

### ISSUE THREAD

**Issue by rcjsuen** [rcjsuen/dockerfile-language-server-nodejs#40]
Originally, `FROM` instructions only took a single argument of the following forms:

1. `FROM <image>`
2. `FROM <image>[:<tag>]`
3. `FROM <image>[@<digest>]`

However, a new `AS` keyword has been introduced and a `FROM` instruction can now be defined by something like `FROM node AS bootstrap`.

As the validator currently validates `FROM` as an instruction that may only have one argument, we need to update it to to accept either one or three arguments (with the second being an `AS`).

**Comment by rcjsuen** [comment]
If a `FROM` instruction does not have one or three arguments, we should flag it as an error.

```
$ docker build .
Sending build context to Docker daemon     83MB
Step 1/1 : FROM alpine AS
FROM requires either one or three arguments
```

**Comment by rcjsuen** [comment]
The message was incorrect but it has been fixed by 124b8d4c339c4b238161b393953ab5dc6e0f7fe7.
